repo_health.rb

```diff
@@ -23,7 +23,7 @@ def check_health(contents, path_key)
 # Github
 user = 'bionode'
 repo = 'bionode'
-data = JSON.parse RestClient.get "https://api.github.com/repos/#{user}/#{repo}/contents", {:params => {:client_id => ENV['ST_GITHUB_CLIENT_ID'], 'client_secret' => ENV['ST_GITHUB_CLIENT_SECRET']}}
+data = JSON.parse RestClient.get "https://api.github.com/repos/#{user}/#{repo}/contents", {:params => {:client_id => ENV['GITHUB_CLIENT_ID'], 'client_secret' => ENV['GITHUB_CLIENT_SECRET']}}
```

Comment:
Line is too long. [190/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.